packages/trader/src/Modules/Contract/Components/AccumulatorsStats/__tests__/accumulators-stats-manual-modal.spec.tsx

```diff
@@ -2,17 +2,27 @@ import React from 'react';
 import { fireEvent, render, screen, within } from '@testing-library/react';
```

Comment:
Do we need to replace `fireEvent` with `userEvent` while refactoring?

Reply:
nice catch